src/compute/src/render/mod.rs

```diff
@@ -524,11 +528,12 @@ where
 
         match bundle.arrangement(&idx.key) {
             Some(ArrangementFlavor::Local(oks, errs)) => {
-                // Set up probes to notify on index frontier advancement.
-                oks.stream.probe_notify_with(probes);
+                // Set up probes to notify on index frontier advancement, and obtain
+                // a specialized handle matching the specialized arrangement.
+                let oks_trace = oks.trace_handle(probes);
```

Comment:
Feels a bit strange to conflate getting a trace handle and attaching probes. Can we make these two methods? E.g.:
```rust
oks.probe_notify_with(probes);
let oks_trace = oks.trace_handle();
```

Reply:
Yep, done!